approval_polls/static/approval_polls/create.js

```diff
@@ -27,7 +29,8 @@ $(function () {
   };
 
   $('[data-toggle=tooltip]').tooltip();
-  $('button#add-choice').click($.proxy(this.addChoiceField, this));
+  $('button#add-choice').click(addChoiceField(this.numChoiceFields));
+  $('button#add-choice-edit').click(addChoiceField(Number($('div#choices-count').text()))
```

Comment:
I don't seem to be able to add a new option as it gives this error:
create.js:33 Uncaught SyntaxError: missing ) after argument list